scene/resources/cubemesh.cpp

```diff
@@ -0,0 +1,396 @@
+/*************************************************************************/
+/*  cube3d.cpp                                                           */
```

Comment:
Lolz, yeah after discussing with reduz we felt renaming this to cubemesh made more sense. The originals were all called something3d

But I'm far more interested in the golden tip that allows me to figure out what I'm doing wrong and why I'm not actually getting a cube created! :)